frameworks/PHP/laravel/laravel-ripple.dockerfile

```diff
@@ -34,15 +34,14 @@ RUN mkdir -p bootstrap/cache \
             storage/framework/cache
 
 # Configure
-RUN echo "PRP_HTTP_LISTEN=http://0.0.0.0:8080" >> .env
-RUN echo "PRP_HTTP_COUNT=64" >> .env
-
 RUN composer install --quiet
-RUN php artisan optimize
-
 RUN composer require cclilshy/p-ripple-drive --quiet
```

Comment:
You can run the require and update in 1 line.
Check : 
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/PHP/laravel/laravel-workerman.dockerfile#L26-L27